test/LibraryManager.IntegrationTest/Microsoft.Web.LibraryManager.IntegrationTest.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net461</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Remove=".editorconfig" />
+    <None Remove="TestSolution\testFile.txt" />
+    <None Remove="TestSolution\WebApplication1\Web.Debug.config" />
+    <None Remove="TestSolution\WebApplication1\Web.Release.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
+    <PackageReference Include="Microsoft.Test.Apex.VisualStudio" Version="15.8.27703" />
+    <PackageReference Include="MSTest.TestAdapter" Version="1.2.1" />
+    <PackageReference Include="MSTest.TestFramework" Version="1.2.1" />
+    <PackageReference Include="Nerdbank.GitVersioning" Version="2.1.23" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="TestSolution\**\*">
+      <LogicalName>TestSolution\%(RecursiveDir)%(Filename)%(Extension)</LogicalName>
+    </EmbeddedResource>
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Remove="TestSolution\WebApplication1\Program.cs" />
```

Comment:
These should be in the grouped globs mentioned above.

Reply:
Will do as compile remove all CSharp files under TestSolution